llvm/lib/Target/AArch64/AArch64ISelLowering.cpp

```diff
@@ -29264,8 +29264,10 @@ void AArch64TargetLowering::verifyTargetSDNode(const SDNode *N) const {
     assert(OpVT.getSizeInBits() == VT.getSizeInBits() &&
            "Expected vectors of equal size!");
     // TODO: Enable assert once bogus creations have been fixed.
```

Comment:
Can this TODO be removed now that the assert has been uncommented?

Reply:
The assert is still disabled for scalable vectors.